src/posix/platform/hdlc_interface.cpp

```diff
@@ -224,7 +224,7 @@ otError HdlcInterface::Write(const uint8_t *aFrame, uint16_t aLength)
 {
     otError error = OT_ERROR_NONE;
 #if OPENTHREAD_POSIX_VIRTUAL_TIME
-    otSimSendRadioSpinelWriteEvent(aFrame, aLength);
+    platformvSendRadioSpinelWriteEvent(aFrame, aLength);
```

Comment:
Current code has defined functions like `RequestV()` and `otCliPlatLogv()`. The character `v` has different means in different functions.  So  if we change the name from `platformvSendRadioSpinelWriteEvent()` to `platformSimSendRadioSpinelWriteEvent()`, it may be better for us to understand. 
Thoughts?

Reply:
updated